code/modules/transport/pods/vehicle.dm

```diff
@@ -36,6 +36,8 @@
 	var/powercapacity = 0 //How much power the ship's components can use, set by engine
 	var/powercurrent = 0 //How much power the components are using
 	var/speed = 1 //FOR PODS : While holding thruster, how much to add on to our max speed. Does nothing for tanks.
+	/// multiplicative ship speed modification from its size
+	var/speedmod = 1
```

Comment:
Having 2 very similarly named fields that do very similar things like this is a footgun for future development, especially with one being a multiplier and one being a divisor.

Reply:
Go to references didn't show all the implementations :/